test/unittests/execute_test.cpp

```diff
@@ -13,9 +13,7 @@ TEST(execute, end)
     (func)
     */
     const auto wasm = from_hex("0061736d01000000010401600000030201000a040102000b");
-    const auto [trap, ret] = execute(parse(wasm), 0, {});
-    ASSERT_FALSE(trap);
-    EXPECT_EQ(ret.size(), 0);
+    EXPECT_EMPTY_RESULT(execute(parse(wasm), 0, {}));
```

Comment:
How about merging `EXPECT_*_RESULT` to expect an array and deduce the size/values from it?

```
EXPECT_RESULT(execute(parse(wasm), 0, {}), {})
EXPECT_RESULT(execute(parse(wasm), 0, {}), {1})
EXPECT_RESULT(execute(parse(wasm), 0, {}), {1, 2})
```

@chfast @gumb0 what od you think?

Reply:
Macros don't go along with `{}`: `EXPECT_RESULT(execute(parse(wasm), 0, {}), {1, 2})` will be parsing error. You need `EXPECT_RESULT(execute(parse(wasm), 0, {}), ({1, 2}))`.
I also don't expect returning all remaining stack will stay with us for long. You may try to get it work with `std::optional<uint64_t>`.